### ISSUE TITLE
Windows/Linux compatibility

### ISSUE THREAD

**Issue by cryptolok** [unixist/cryptostalker#7]
Are you sure that :
`52:    if time.Since(i.CreationTime()) < age {`
is compatible with Linux ?
Since CreationTime [belongs](https://golang.org/src/os/exec_windows.go?h=CreationTime) to Windows, I was unable to launch the executable correctly on Linux (Debian 8 x64), because I'm getting 2562047 hours...

**Comment by cryptolok** [comment]
In My Hackish Opinion, it will be simpler to kill the last process (or few more), since we're in a loop :
`ret = append(ret, procs[len(procs)-1].Pid()`
Actually, it opens-up a way to bypass the detection, but it's possible anyway

**Comment by unixist** [comment]
The CreationTime() function comes from [here](https://github.com/unixist/go-ps/blob/master/process.go#L28). I added it in my fork of [mitchellh/go-ps](https://github.com/mitchellh/go-ps) since that package doesn't contain process time information.

I only populate CreationTime() on Windows for now. So yes :( this feature is broken on anything non-Windows. I'd happily accept a PR to add support in [unixist/go-ps](github.com/unixist/go-ps) for process times in Linux (and OSX if you're feeling ambitious!). I created issues #8 and #9 for these.

This will allow new issues #10 and #11 to support sending SIGSTOP to recent processes in Linux and OSX. This is much better than just killing the processes in Windows (which must happen since the go library doesn't support stopping processes in Windows, just killing them).
